planning/behavior_path_planner/autoware_behavior_path_lane_change_module/README.md

```diff
@@ -304,53 +411,65 @@ if (Check if start point is valid by check if it is covered by neighbour lanes p
 else (covered)
 endif
 
-group check for distance #LightYellow
-    :Calculate total length and goal related distances;
-    if (total lane change length considering single lane change > distance from current pose to end of current lanes) then (yes)
-      #LightPink:Reject path;
-      stop
-    else (no)
-    endif
-
-    if (goal is in current lanes) then (yes)
-      if (total lane change length considering multiple lane changes > distance from ego to goal along current lanes) then (yes)
-         #LightPink:Reject path;
-        stop
-      else (no)
-      endif
-    else (no)
-    endif
+:Calculate total length and goal related distances;
+if (total lane change length considering single lane change > distance from current pose to end of current lanes) then (yes)
+  #LightPink:Reject path;
+  stop
+else (no)
+endif
 
-    if (target lanes is empty) then (yes)
+if (goal is in current lanes) then (yes)
+  if (total lane change length considering multiple lane changes > distance from ego to goal along current lanes) then (yes)
       #LightPink:Reject path;
-      stop
-    else (no)
-    endif
-    if (total lane change length considering multiple lane changes  > distance from ego to the end of target lanes) then (yes)
-      #LightPink:Reject path;
-      stop
-    else (no)
-    endif
-end group
+    stop
+  else (no)
+  endif
+else (no)
+endif
 
-if (ego is not stuck but parked vehicle exists in target lane) then (yes)
+if (target lanes is empty) then (yes)
   #LightPink:Reject path;
   stop
 else (no)
 endif
-
-if (is safe to perform lane change) then (yes)
-  #Cyan:Return candidate path list;
+if (total lane change length considering multiple lane changes  > distance from ego to the end of target lanes) then (yes)
+  #LightPink:Reject path;
   stop
 else (no)
-  #LightPink:Reject path;
 endif
 
+#LightGreen:Valid Candidate Path;
 stop
 
 @enduml
 ```
 
+!!! warning
+
+    A valid path does NOT mean that the path is safe, however it will be available as a candidate path and can be force approved by operator. A path needs to be both valid AND safe to be automatically approved.
+
+#### Candidate Path Safety
+
+A candidate path is considered safe if:
+
+1. There is no overtaking turn lane object (see Overtaking Object Check section below)
```

Comment:
The closest explanation is "there's no overtaking object when ego vehicle exited turn direction lane"